### DISCUSSION TITLE
Problems with Coreshop 4.x installation

### DISCUSSION THREAD

**Discussion by twin-elements** [coreshop/CoreShop#2366] [Q&A] [answered]
Hello.
I installed a clean version of Pimcore11.  Everything works as expected.

Then I installed Coreshop 4 according to the instructions:
`composer require coreshop/core-shop:^4.0`

All Coreshop packages can be found in the list of packages:
![image](https://github.com/coreshop/CoreShop/assets/17727266/ec0906c3-1820-4191-8368-45c18cceca15)

When I enable CoreShopCoreBundle in bundles.php I get an error
`You have requested a non-existent service "pimcore.cache.adapter.redis_tag_aware".`

![image](https://github.com/coreshop/CoreShop/assets/17727266/3e907b45-4a1a-4624-908d-484796066677)

`bundles.php` file
![image](https://github.com/coreshop/CoreShop/assets/17727266/bf6fff8b-aef1-4d6a-8410-f2188c8ae2e9)

This error blocks further installation of Coreshop 4. I am very much asking for help, as the documentation is outdated at the moment.

Thanks.

**Comment by dpfaffenbauer** [comment, marked as answer]
@twin-elements Pimcore introduced a BC break by changing a DI Definition to an Alias (https://github.com/pimcore/pimcore/pull/15625), which caused a changed on our side: https://github.com/coreshop/CoreShop/pull/2362. This change is not included in the alpha that you probably installed, so install `coreshop/core-shop:^4.0.x-dev`

**Reply by twin-elements** [reply]
Thank you for help!
